### DISCUSSION TITLE
getFeedSkeleton 'feedContext' usage

### DISCUSSION THREAD

**Discussion by hecksadecimal** [bluesky-social/atproto#3114] [Bluesky Lexicons]
I'm trying to build [a feed that identifies and extracts 'accidental' haikus](https://github.com/hecksadecimal/haiku-feed).
I'd like to use feedContext on individual posts returned in the feed to give feed viewers a way to see how the algo interpreted a haiku from the post's contents, instead of having a bot go around replying to everybody. 

The [documentation](https://docs.bsky.app/docs/api/app-bsky-feed-get-feed-skeleton) seems to imply this is the correct use of feedContext, but I cannot get my added context to reflect on bsky.app. 

Is there something I'm missing? Has anyone else successfully used feedContext? Are there examples?

**Comment by mary-ext** [comment]
feedContext isn't a value shown to the user, it's used as part of feed interactions API which sends feedback about how users are interacting with the feed.

Currently only Discover can make use of that functionality.

**Reply by hecksadecimal** [reply]
Thanks for this explanation. I wish that this fact was better documented/worded, I've been chasing a red herring this whole time.
